package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "rm -rf client/ dev/ && babel src/ --out-dir ./ && chmod +x dev/cli/*.js",
     "prepublish": "npm run build",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest src/ --coverage"
```

Comment:
What about moving tests to a separate folder (`/tests/`)? I think right now they are also transpiled and included in the package (`npm pack && tar tzf react-jsx-i18n-0.0.1.tgz` to check).